blocks/cards/cards.css

```diff
@@ -1,27 +1,187 @@
+    padding: var(--qui-spacing-16) var(--qui-spacing-0) var(--qui-spacing-0) var(--qui-spacing-0) !important;
 }
 
 .cards > ul > li img {
-  width: 100%;
-  aspect-ratio: 4 / 3;
-  object-fit: cover;
+    transform: scale(2.66);
+    box-sizing: border-box;
+    margin-left: var(--qui-spacing-20);
+    margin-bottom: var(--qui-spacing-40);
+}
+
+.cards .card-image {
+    padding-top: 44px !important;
+    padding-bottom: var(--qui-spacing-4) !important;
+    padding-left: var(--qui-spacing-0);
+}
+
+.cards .card-container {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+}
+
+.cards .card-title {
+    padding-bottom: var(--qui-spacing-16);
+    margin-bottom: var(--qui-spacing-0);
+}
+
+.union-palette .cards .card-container,
+.cards.card-container.union-palette {
+    padding-top: var(--qui-spacing-16) !important;
+}
+
+.cards .card-container:first-child {
+    padding-top: var(--qui-spacing-24) !important;
 }
+
+.cards .card-text {
+    margin-bottom: var(--qui-spacing-0);
+}
+
+.cards .card-button {
+    padding-top: var(--qui-spacing-16);
+    padding-bottom: var(--qui-spacing-16);
+}
+
+.union-palette .cards .card-button,
+.cards.union-palette .card-button{
+    margin-bottom: var(--qui-spacing-0);
+}
+
+@media (width >= 600px) and ( width <= 959px) {
+    .union-palette .cards > ul > li,
+    .cards.union-palette  > ul > li {
+        padding: var(--qui-spacing-0) var(--qui-spacing-0) var(--qui-spacing-0) var(--qui-spacing-0) !important;
+    }
+
+    .union-palette .cards .card-container,
+    .cards.union-palette .card-container {
+        padding-top: var(--qui-spacing-0);
+        padding-bottom: var(--qui-spacing-0);
+    }
+
+    .union-palette .cards .card-button,
+    .cards.union-palette .card-button {
+        padding-top: var(--qui-spacing-16);
+        padding-bottom: var(--qui-spacing-32);
+    }
+
+
+    .union-palette .cards .card-title,
+    .cards.union-palette .card-title {
+        padding-top: var(--qui-spacing-24);
+        padding-bottom: var(--qui-spacing-16);
+    }
+}
+
+@media (width >= 960px) {
+    .cards > ul {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+        display: grid;
+        grid-template-rows: repeat(var(--bs-rows, 1), 1fr);
+        grid-template-columns: repeat(var(--bs-columns, 12), 1fr);
+        gap: var(--qui-spacing-32);
+        padding-top: var(--qui-spacing-40);
+    }
+
+    .union-palette .cards > ul,
+    .cards.union-palette  > ul {
+        padding-top: var(--qui-spacing-0);
+    }
+
+    .cards > ul > li {
+        box-sizing: border-box;
+        display: flex;
+        border-left: var(--qui-alias-border-width-thin) solid var(--qui-color-neutral-gray-300);
+        border-top: none;
+        flex-direction: column;
+        grid-column: auto / span 4;
+        padding-left: var(--qui-spacing-24);
+    }
+
+    .union-palette .cards > ul > li,
+    .cards.union-palette  > ul > li {
+        grid-column-end: span 5;
```

Comment:
One last ask, please: I understand that the cards that are right after the hero have some custom spacing (like this `span 5`). For these styles we should introduce a variant, so those cards should be something like `Cards (Hero Subcards)` and then have the CSS which is specific for that as `.cards.hero-subcards`.

The reason is: Not all cards which are created under a union-pallete should have this custom styling. The pallete class is just a colouring scheme. Authors could change the colouring scheme of the "count on us" section anytime to be union palette if they wish. This shouldn't change the spacing of those cards. The custom spacing belongs just to those hero subcards.

So things like the custom column span should go under `.cards.hero-subcards`.
Things like a specific border colour for cards in the union palette should remain under `.cards.union-palette`.

Reply:
Fixed